### PR TITLE
[RM Ch04] Integrate New PM/Cap Identifier Schema +

### PR DESCRIPTION
**Pull request by markshostak**
- Integrate New PM/Cap Identifier Schema // Issue #390
- Remove PMs relocated to RCx/RMCH-8 // Issue #378
- Add draft definitions in support of schema // currently commented out
- Mechanical and typo cleanup

### CODE REVIEW

File: doc/ref_model/chapters/chapter04.md [hunk 1]
```diff
@@ -75,13 +88,13 @@ This section describes a set of explicit NFVI capabilities and performance measu
...
-| e.nfvi.per.cap.005 | Transcoding Acceleration | Yes/No | Transcoding Acceleration |
-| e.nfvi.per.cap.006 | Programmable Acceleration | Yes/No | Programmable Acceleration |
-| e.nfvi.per.cap.007 | Enhanced Cache Management* | enum | L=Lean; E=Equal; X=eXpanded |
+| e.nfvi.cap.006 | CPU core pinning support | Yes/No | Indicates if NFVI supports CPU core pinning |
+| e.nfvi.cap.007 | NUMA support | Yes/No | Indicates if NFVI supports NUMA |
+| e.nfvi.cap.008 | IPSec Acceleration | Yes/No | IPSec Acceleration |
+| e.nfvi.cap.009 | Crypto Acceleration | Yes/No | Crypto Acceleration |
+| e.nfvi.cap.010 | Transcoding Acceleration | Yes/No | Transcoding Acceleration |
+| e.nfvi.cap.011 | Programmable Acceleration | Yes/No | Programmable Acceleration |
+| e.nfvi.cap.012 | Enhanced Cache Management* | Yes/No | If supported, L=Lean; E=Equal; X=eXpanded |
```
Comment (ulikleber) [inline doc/ref_model/chapters/chapter04.md]: small question: How do we express L/E/X capability with a yes/no Unit? Shouldn't it be enum with L=Lean; E=Equal; X=eXpanded and N=NotSupported?

**Comment by xavier-grall** [comment]
I guess the "unit" should rather be: No/Lean/Equal/eXpanded

**Comment by markshostak** [comment]
@trevgc, @karinesevilla can you approve this PR pls? Merci beaucoup, -Mark

File: doc/ref_model/chapters/chapter04.md (hunk 1 above)
Comment (markshostak) [reply doc/ref_model/chapters/chapter04.md]: I'm glad I'm not the only one who thought that! That was the logic I used originally, but the datum is actually supposed to represent whether or not the capability to control the cache _is present_ in the give infra, which it either is or it is not, hence the flag is typed as a bool. When you configure it, if it's supported, you would use the enum (L/E/X). Thx, -Mark

**Comment by markshostak** [comment]
> 
> 
> @markshostak
> Subsection 4.1.2.1, following paragraph must be moved to chapter 8
> "It should be noted that energy-related metrics must only be considered for NFVI software running on a same NFVI hardware implementation (since energy consumption may be very different across processor architecures, models and power management features)."
> The paragraph refers to content no more there

Bonjour Karine, I've opened Issue #560 to address this, but unless you have additional concerns, I'd like to go ahead and merge this PR. Merci, -Mark

**Comment by xavier-grall** [comment]
@markshostak @karinesevilla 
Since PR #464 (from issue #462) includes the move suggested by Karine, I think you can reasonnably remove that text at once.

**Comment by karinesevilla** [comment]
@markshostak @xavier-grall 
Yes I agree with Xavier, just remove the paragraph, it will be sufficient

**Comment by markshostak** [comment]
> 
> 
> @markshostak @xavier-grall
> Yes I agree with Xavier, just remove the paragraph, it will be sufficient

@karinesevilla I removed that language, but found it under 4.1.5.1. Is this the correct place? Marci, -Mark

**Comment by karinesevilla** [comment]
@markshostak Yes you're right, my mistake, it's 4.1.5.1

**Pull request merged**